src/workerd/api/r2-bucket.h

```diff
@@ -258,6 +273,9 @@ class R2Bucket: public jsg::Object {
     jsg::Optional<kj::Array<jsg::NonCoercible<kj::String>>> include;
 
     JSG_STRUCT(limit, prefix, cursor, delimiter, startAfter, include);
+    JSG_STRUCT_TS_OVERRIDE(R2ListOptions {
+      include?: ("httpMetadata" | "customMetadata")[];
```

Comment:
I think this should also have an override based on [#r2-bucket-list-respects-the-include-option](https://developers.cloudflare.com/workers/platform/compatibility-dates/#r2-bucket-list-respects-the-include-option) that sets it to `never` without this flag